openreview/conference/builder.py

```diff
@@ -1477,62 +1477,123 @@ def get_attachment_file(submission):
-                                    url_forum=submission.forum,
-                                    accepted=note_accepted,
-                                    anonymous=(not release_authors))
-                    }
-                    if not release_authors:
-                        submission.content['authors'] = ['Anonymous']
-                        submission.content['authorids'] = [self.get_authors_id(number=submission.number)]
-                else:
-                    submission.content['_bibtex'] = tools.get_bibtex(
-                                    submission,
-                                    venue_fullname=self.name,
-                                    year=str(self.year),
-                                    url_forum=submission.forum,
-                                    accepted=note_accepted,
-                                    anonymous=False)
-                if note_accepted:
-                    decision = decision_note.content['decision'].replace('Accept', '')
-                    decision = re.sub(r'[()\W]+', '', decision)
-                    venueid = self.id
-                    venue = self.short_name
-                    if decision:
-                        venue += ' ' + decision
-                    submission.content['venueid'] = venueid
-                    submission.content['venue'] = venue
-                self.client.post_note(submission)
+            if hide_rejected:
+                submission.readers = self.submission_stage.get_readers(self, submission.number, decision_note)
+            else:
+                submission.readers = self.submission_stage.get_readers(self, submission.number)
+            #double-blind
+            if self.submission_stage.double_blind:
+                release_authors = is_release_authors(note_accepted)
+                submission.content = {
+                    '_bibtex': tools.get_bibtex(
+                        openreview.Note.from_json(submission.details['original']),
+                        venue_fullname=self.name,
+                        year=str(self.year),
+                        url_forum=submission.forum,
+                        accepted=note_accepted,
+                        anonymous=(not release_authors)
+                    )
+                }
+                if not release_authors:
+                    submission.content['authors'] = ['Anonymous']
+                    submission.content['authorids'] = [self.get_authors_id(number=submission.number)]
+            #single-blind
+            else:
+                submission.content['_bibtex'] = tools.get_bibtex(
+                    submission,
+                    venue_fullname=self.name,
+                    year=str(self.year),
+                    url_forum=submission.forum,
+                    accepted=note_accepted,
+                    anonymous=False
+                )
+            #add venue_id if note accepted
+            if note_accepted:
```

Comment:
I think we should add a `venue_id` for rejected papers like `Submitted to {venue_id}`. What do you think?

Reply:
Done